IPython/utils/terminal.py

```diff
@@ -19,7 +19,10 @@
     from shutil import get_terminal_size as _get_terminal_size
 except ImportError:
     # use backport on Python 2
-    from backports.shutil_get_terminal_size import get_terminal_size as _get_terminal_size
+    try:
+        from backports.shutil_get_terminal_size import get_terminal_size as _get_terminal_size
+    except ImportError:
+        from _get_terminal_size import _get_terminal_size
```

Comment:
Let's put the `.` in for an explicit relative import. It's clearer, and if someone adds `from __future__ import absolute_import`, it won't break.